### PR TITLE
Generate Draco and Time Dynamic Point Cloud tilesets

### PR DESCRIPTION
**Pull request by lilleyse**
Generates sample tilesets for https://github.com/AnalyticalGraphicsInc/cesium/pull/6721 and https://github.com/AnalyticalGraphicsInc/cesium/pull/6559

### CODE REVIEW

File: samples-generator/bin/3d-tiles-samples-generator.js [hunk 1]
```diff
@@ -956,11 +1014,14 @@ function savePointCloudTileset(tilesetName, tileOptions, tilesetOptions) {
     var result = createPointCloudTile(tileOptions);
     var pnts = result.pnts;
     var batchTableJson = result.batchTableJson;
+    var extensionsUsed = result.extensionsUsed;
```
Comment (ggetz) [inline samples-generator/bin/3d-tiles-samples-generator.js]: I added an `Extensions` class in https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/pull/149/files, do you want to use that?

File: samples-generator/lib/createPointCloudTile.js [hunk 2]
```diff
@@ -175,7 +232,175 @@ function createPointCloudTile(options) {
...
+
+    if (preserveOrder) {
+        encoder.SetEncodingMethod(encoderModule.POINT_CLOUD_SEQUENTIAL_ENCODING);
+    }
+
+    var encodedDracoDataArray = new encoderModule.DracoInt8Array();
+
+    var encodedLength = encoder.EncodePointCloudToDracoBuffer(pointCloud, false, encodedDracoDataArray);
+    if (encodedLength <= 0) {
+        throw 'Error: Encoding Failed.';
```
Comment (ggetz) [inline samples-generator/lib/createPointCloudTile.js]: `Draco encoding failed`? Just so there's context if someone gets this error.

**Comment by lilleyse** [comment]
@ggetz this is finally ready

**Comment by ggetz** [comment]
Thanks @lilleyse!

**Pull request merged**